src/main/jbake/content/blog/2017/07/grafana-new-query-interface.adoc

```diff
@@ -0,0 +1,177 @@
+= Grafana: new query interface
+Joel Takvorian
+2017-07-20
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, metrics, grafana
+:figure-caption!:
+
+There has been several improvements lately in our Hawkular Grafana datasource, that is worth writing some words about.
+
+First, the way to query metrics by tags has changed since plugin link:https://github.com/hawkular/hawkular-grafana-datasource/releases/tag/v1.0.8[v1.0.8].
+It now takes advantage of the Hawkular Metrics' tags query language,
+that was introduced server-side in link:http://www.hawkular.org/blog/2017/02/08/hawkular-metrics-0.24.0.Final-released.html[Metrics 0.24.0]
+and enhanced in link:http://www.hawkular.org/blog/2017/03/07/hawkular-metrics-0.25.0.Final-released.html[0.25.0].
+To sum it up, Metrics integrates a parser that allows queries such as:
+`tag1 = 'awesome!' AND tag2 NOT IN ['foo', 'bar']`.
+
+Secondly, the datasource in now able to fetch bucketized metrics stats, instead of raw metrics. Hawkular has always been able to provide them server-side,
+but having it used in the Grafana plugin is new, introduced in link:https://github.com/hawkular/hawkular-grafana-datasource/releases/tag/v1.0.9[v1.0.9].
+
+[.text-center]
+ifndef::env-github[]
+image::/img/blog/2017/2017-07-20-grafana-empty-query.png[New query interface]
+endif::[]
+ifdef::env-github[]
+image::../../../../assets/img/blog/2017/2017-07-20-grafana-empty-query.png[New query interface]
+endif::[]
+image::../../../../assets/img/blog/2017/2017-07-20-grafana-empty-query.png[New query interface]
+
+[.text-center]
```

Comment:
That does not render correctly - what are you trying to achieve?

If you are trying to put titles/captions on the images, then have a look at http://asciidoctor.org/docs/user-manual/#images, callout number 2